noxfile.py

```diff
@@ -318,6 +318,7 @@ def tests(session: nox.Session) -> None:
 
     args = ["-m", "pytest"]
     args += coverage_args
+    args += ["--ignore-glob", "*/external/bazel_tools/*"]
```

Comment:
Possibly everything in external should be ignored?

Reply:
I think `external` is a too generic exclude pattern.

Reply:
I'm not sure what directory the search is starting from but it I think it may be the external dependencies pulled in by Bazel. Do you need to access those for external third party libraries?

Reply:
I don't need this directory symlinked to my build directory.